addons/web/static/src/views/fields/copy_clipboard/copy_clipboard_field.js

```diff
@@ -79,7 +87,10 @@ function extractProps({ string, attrs }) {
 export const copyClipboardButtonField = {
     component: CopyClipboardButtonField,
     displayName: _t("Copy to Clipboard"),
-    extractProps,
+    extractProps: ({ string, attrs }) => ({
+        ...extractProps({ string, attrs }),
+        btn_class: attrs.btn_class,
```

Comment:
```suggestion
    extractProps: (fieldInfo) => ({
        ...extractProps(fieldInfo),
        btnClass: fieldInfo.options.btn_class,
```

Hello :-)
- btn_class is set as option and not as direct attribute, as it is an option, and the list of attribute must be kept minimal
- btn_class => btnClass as by convention, variables are camelcased in JS
- this new option should be tested